src/SapAct/ADXWorker.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace SapAct;
+
+public class ADXWorker(ServiceBusClient sbClient, ADXService adxService, ILogger<ADXWorker> logger) : BackgroundService
+{
+	private ServiceBusReceiver? serviceBusReceiver;
+
+	protected override async Task ExecuteAsync(CancellationToken stoppingToken)
+	{
+		serviceBusReceiver = sbClient.CreateReceiver("sap-events", "SDV", new ServiceBusReceiverOptions() //TODO: make topic subscription name configurable
```

Comment:
suggestion... instead of a TODO, why don't you just defer this to the appsettings or something?

Reply:
because I need to have multiples... so it will be tied to worker type...need to create convention and actually agree with topic owner on this